src/isysmeta.cpp

```diff
@@ -494,7 +494,7 @@ usage() {
         " ls [-lvV] Name (list dataObject, -l -v for long form)",
         " ldt (list data types (those available))",
         " ",
-        "Time can be full or partial date/time: '2009-12-01' or '2009-12-11.12:03'",
+        "Time can be full or partial date/time: '2024-12-01' or '2024-12-11.12:03'",
```

Comment:
The other date replacements say 2025

Reply:
yes, b/c dec 2025 hasn't happened yet.    open to suggestions - this was the least work :)

Reply:
Well, that makes a lot of sense. Resolving.